src/webfaf/static/css/style.css

```diff
@@ -1447,7 +1447,7 @@ td > div.progress {
   margin-bottom: 0;
 }
 
-tr.odd, tr.stripe {
```

Comment:
I was going to suggest going for `:nth-child(odd)`, but you would probably have to grab your can opener for the cans of spaghetti that would unleash.

Reply:
Yup, it's unfortunately not that easy.